tests/batch-execute-query-test.bal

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to have all test cases here, only connection.bal test cases are enough. The point of test cases is to identify the driver is working thats all.

Reply:
Fixed in 2f32c83